site/sk.njk

```diff
@@ -0,0 +1,161 @@
+---
+layout: default
+meta_title: Hnutie za zastavenie pandémie COVID-19
+meta_description: Naše vlády to nemôžu urobiť sami, musíme konať teraz! Urobte svoju časť a zostante doriti doma!
+body_class: home
+title: '#StayTheFuckHome'
+lang: sk
+---
+
+<main id="main" class="main inner flex flex-1 flex-col py-10 lg:py-20 focus:outline-none" tabindex="-1">
+
+    <section id="container-centre" class="column centre flex-1">
+
+        <h1 class="page-title">{{ title }}</h1>
+        <h2 class="page-subtitle">Hnutie za zastavenie pandémie COVID-19</h2>
+
+        <div class="content">
+
+            <p class="text-xl">
+                Naše vlády len pomaly implementujú opatrenia na kontrolu šírenia
+                <a href="https://sk.wikipedia.org/wiki/SARS-CoV-2">SARS-CoV-2</a> a obsahujúci
+                <a href="https://sk.wikipedia.org/wiki/COVID-19">COVID-19</a>. Pomalé reakcie politiky upokojovania
+                verejnosti a ich snaha o stabilizáciu ekonomiky im zabránila brať vážne ich opatrenia potrebné na
+                ochranu miliónov pred touto chorobou. Avšak, toto bremeno nie je len na vláde. Je čas, aby sme my ako
```

Comment:
> Avšak, 
  
Sounds really odd at the beginning of the sentence. An obvious literal translation of "However, ".